team-agreements/definition-of-ready/readme.md

```diff
@@ -2,20 +2,24 @@
 
 When the development team picks a user story from the top of the backlog, the user story needs to have enough detail to estimate the work needed to complete the story within the sprint. If it has enough detail to estimate, it is Ready to be developed.
 
-> In case a user story is not Ready in the beginning of the Sprint it increases the chance that your story will not be done at the end of this sprint.
+> If a user story is not Ready in the beginning of the Sprint it increases the chance that the story will not be done at the end of this sprint.
 
 ## What it is
 
-Definition of Ready is the agreement made by the scrum team around how complete a user story should be in order to be selected as candidate for estimation in the sprint planning.
+*Definition of Ready* is the agreement made by the scrum team around how complete a user story should be in order to be selected as candidate for estimation in the sprint planning. These can be codified as a checklist in user stories using [Github Issue Templates](https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository) or [Azure DevOps Work Item Templates](https://docs.microsoft.com/en-us/azure/devops/boards/backlogs/work-item-template?view=azure-devops&tabs=browser).
 
-It can be understood as a checklist that helps the Product Owner to ensure that the user story he wrote contains all the necessary details for the scrum team to understand the work to be done.
+It can be understood as a checklist that helps the Product Owner to ensure that the user story they wrote contains all the necessary details for the scrum team to understand the work to be done.
 
 ### Examples of ready checklist items:
 
 * [ ] Does the description have the details including any input values required to implement the user story?
 * [ ] Does the user story have clear and complete acceptance criteria?
 * [ ] Does the user story address the business need?
 * [ ] Can we measure the acceptance criteria?
+* [ ] Can the story be completed within a single sprint? If not, can it be broken down into smaller stories?
+* [ ] Is the story blocked? For example, does it depend on any of the following:

```

Comment:
These are important examples to include in the checklist, thank you. Only a minor reminder to replace "story" by "user story" to keep consistency across the document.